source/support.html.erb

```diff
@@ -48,9 +48,7 @@ title: Support – GOV.UK Verify
             if it's a known issue with the platform, it will show there and we'll keep our users updated on how the incident is progressing
             </li>
             <li>
-              if there are no incidents, email
-              <a href="mailto:idasupport@digital.cabinet-office.gov.uk" class="govuk-link">idasupport@digital.cabinet-office.gov.uk</a>
-              with the details of the problem your service is experiencing
+              if there are no incidents, email our support team with the details of the problem your service is experiencing
```

Comment:
This feels wrong: well-run services teams might have avoided writing down the email address, and will have referenced either this page, or 
a page that links to it, in their team manual. We should still make it easy for existing services to get in touch via email, without going via the end-user support team.

Reply:
We can email all services with an FYI message to save our support email address somewhere. Leaving it on a public page will defeat the purpose of making these changes and allow people to continue emailing us with random queries and bloating our 2nd line.

Reply:
Agreed to move this content to self-service and hide behind a log-in page